packages/react-native-gesture-handler/jestSetup.js

```diff
@@ -1,7 +1,16 @@
 jest.mock('./src/RNGestureHandlerModule', () => require('./src/mocks'));
+jest.mock('./src/components/GestureButtons', () => require('./src/__mocks__/GestureButtons'));
+
 jest.mock('./lib/commonjs/RNGestureHandlerModule', () =>
   require('./lib/commonjs/mocks')
 );
+jest.mock('./lib/commonjs/components/GestureButtons', () =>
+  require('./lib/commonjs/mocks')
```

Comment:
As far as I can see `bob` doesn't generate `__mocks__` directory when running `yarn build`. That's ok, but does this actually work, or does it work by accident because this file already contains these mocks?

What I want to say - do we need `GestureButtons` mock file?